src/Init/Data/Int/Pow.lean

```diff
@@ -56,4 +56,11 @@ protected theorem two_pow_pred_sub_two_pow' {w : Nat} (h : 0 < w) :
   rw [← Nat.two_pow_pred_add_two_pow_pred h]
   simp [h]
 
+theorem pow_lt_pow {a : Int} {b c : Nat} (ha : (1 : Nat) < a) (hbc : b < c):
```

Comment:
I'm not sure about normal forms here: I would imagine we want `ha : 1 < a`, so I'm a bit surprised to see `(1 : Nat) < a`.